account_banking_pain_base/views/account_pain_regulatory_reporting.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+  Copyright 2023 Akretion France (http://www.akretion.com/)
+  @author: Alexis de Lattre <alexis.delattre@akretion.com>
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+
+<record id="account_pain_regulatory_reporting_form" model="ir.ui.view">
+    <field name="model">account.pain.regulatory.reporting</field>
+    <field name="arch" type="xml">
+        <form>
+            <widget
+                    name="web_ribbon"
+                    title="Archived"
+                    bg_color="bg-danger"
+                    attrs="{'invisible': [('active', '=', True)]}"
+                />
+            <group name="main">
+                <field name="code" />
+                <field name="name" />
+                <field name="country_id" />
+                <field name="active" invisible="1" />
+        </group>
+</form>
```

Comment:
Incorrect indentation

Reply:
I let prettier do the formatting work and pre-commit doesn't give any error. Strange !